css/faq.css

```diff
@@ -159,143 +191,152 @@ h3 > a > i.fa-chevron-left {
     font-weight: bold;
     line-height: 30px;
 }
-.tiles-container.faq-tiles .tile:hover > .tile-head {
+
+.tiles-container.faq-tiles .tile:hover>.tile-head {
     background: #d0cdcd;
 }
-.tiles-container.faq-tiles .tile > .tile-head > span {
+
+.tiles-container.faq-tiles .tile>.tile-head>span {
     padding: 7px 15px;
     margin-right: 15px;
     border-right: 1px solid #BBB;
     display: flex;
     align-items: center;
 }
-.tiles-container.faq-tiles .tile > .tile-head > span.tutorial {
+
+.tiles-container.faq-tiles .tile>.tile-head>span.tutorial {
     padding: 7px 12px 7px 11px;
 }
-.tiles-container.faq-tiles .tile > .tile-head > strong {
+
+.tiles-container.faq-tiles .tile>.tile-head>strong {
     padding: 7px 15px;
     width: calc(100% - 45px);
 }
-.tiles-container.faq-tiles .tile > .tile-body {
+
+.tiles-container.faq-tiles .tile>.tile-body {
     display: none;
-    #height: 0;
```

Comment:
Rules which have a hash in front of them will not be applied and thus can be removed.

Reply:
ah, okay.
I was not sure if this is a typo or some old browser hack.
Will remove them :+1:

Reply:
Rules with a hash are of course _tried_ to be interpreted. However, this is invalid syntax in CSS, and therefore the line will be ignored. There is no commenting syntax in CSS.